src/main/java/appeng/api/storage/cells/StorageCell.java

```diff
@@ -39,6 +39,11 @@ public interface StorageCell extends MEStorage {
      */
     double getIdleDrain();
 
+    /**
+     * Return whether the cell should be storable in other kinds of cells.
+     */
+    boolean canFitInsideCell();
```

Comment:
Needs a default implementation returning true to preserve backwards compatibility. (This is an API class with foreign implementations).